src/app/app.module.ts

```diff
@@ -3,10 +3,14 @@ import { BrowserModule } from '@angular/platform-browser';
 
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
+import { LoginComponent } from './components/login/login.component';
+import { RegisterComponent } from './components/register/register.component';
 
 @NgModule({
   declarations: [
-    AppComponent
+    AppComponent,
+    LoginComponent,
+    RegisterComponent
```

Comment:
It's time to learn a key concept in Angular called [LazyLoading](https://angular.io/guide/lazy-loading-ngmodules). This should be explained on the heroes course.